src/mainboard/clevo/tgl-u/ramstage.c

```diff
@@ -31,13 +31,30 @@ struct smfi_cmd_set_fan_curve {
 	struct fan_curve curve;
 };
 
-#define FAN_CURVE_DEFAULT {     \
+#define FAN_CURVE_SILENT {      \
   { .temp = 0,   .duty = 25  }, \
```

Comment:
coreboot uses TAB indents (8 character long))

Reply:
ofc, fixed, copypaste error when moving this code from DasharoModulePkg